### DISCUSSION TITLE
RPM Pulsar not compatible with Atom

### DISCUSSION THREAD

**Discussion by mramendi** [pulsar-edit/.github#88] [Pulsar Support]
I have Fedora 36, with Atom installed from its official RPM. As Atom is going EOL I downloaded the Pulsar RPM and tried to install it. Here is what I got:

```
Error: Transaction test error:
  file /usr/lib/.build-id/3b/04b76eeceabfa7eb89531adce2912e952fcde1 from install of pulsar-1.63.2022121415-1.x86_64 conflicts with file from package atom-1.60.0-0.1.x86_64
  file /usr/lib/.build-id/53/3c7d4053e1bf666783925a192552c0052b11d9 from install of pulsar-1.63.2022121415-1.x86_64 conflicts with file from package atom-1.60.0-0.1.x86_64
```

Is this intended?

**Comment by Spiker985** [comment]
Nope, not at all. Does it contain any additional information regarding the conflict?

**Reply by Spiker985** [reply]
Can I also get you to make an issue for this within the Pulsar repo?

**Reply by Daeraxa** [reply]
We should be able to transfer https://github.com/pulsar-edit/.github/issues/89

EDIT: Now on https://github.com/pulsar-edit/pulsar/issues/227

**Reply by mramendi** [reply]
Thank you! I have added rpm -vv output to the issue.
